src/Microsoft.SqlTools.ServiceLayer/HostLoader.cs

```diff
@@ -165,6 +165,9 @@ private static void InitializeRequestHandlersAndServices(ServiceHost serviceHost
             InitializeHostedServices(serviceProvider, serviceHost);
             serviceHost.ServiceProvider = serviceProvider;
 
+            ShowPlan.ExecutionPlanService.Instance.InitializeService(serviceHost);
```

Comment:
nit: "ShowPlan" is more of an internal SQL engine name, probably QueryExecutionPlan or ExecutionPlan is a better name.

Reply:
Will rename that in a separate PR. That will require a lot of file name changes.